vendor/vendor.json

```diff
@@ -3879,10 +3912,18 @@
 			"revisionTime": "2019-08-23T10:51:29Z"
 		},
 		{
-			"checksumSHA1": "MA07KaAp1aPVheuopyrFr7pxANs=",
+			"checksumSHA1": "SInqynLK8NjJIeFs0Mtk4g8LJ+A=",
+			"path": "github.com/opencontainers/runc/libcontainer/system",
+			"revision": "2b52db75279ca687e18156de86d845876e9ef35d",
```

Comment:
Could you please use the version `v1.0.0-rc9`, same as for `github.com/opencontainers/runc/libcontainer/user`?